loop.input

```diff
@@ -0,0 +1,42 @@
+nput parameters for cvc-code
```

Comment:
can you delete this file please?

Reply:
What is the problem with this file?

Reply:
it's sitting in the main directory  and there's a copy of it in the ```example_input_files``` directory that you created

Reply:
oh, sorry, I am doing it.